### PR TITLE
Make assets page responsive

### LINKED ISSUE #17: Update Assets page to fit on mobile
As a user I want to see full assets page when I’m on my mobile.

# Replication steps 

1. Click 'Assets'  in mobile view

The assets page has poor formatting. Please update to make it match the attached design.

Please ensure: 

- All the columns can be seen, including: (symbol, volume, decimal,  type).

- The items in the ‘Type’ dropdown are visible.

Currently a different ‘Asset’ and/or ‘Type’ dropdown item can be set and the results can be filtered when the ‘Filter’ button is clicked.

# Prerequisites

You will need to know basic Javascript (jQuery) and css (scss) to do this task.

# Resources

## Repository link

https://github.com/cennznet/explorer/tree/master 

This ‘Explorer’ repository contains several projects. 

The project you need to work on is in the ‘portal’ directory.

# Design

Please refer to the InVision prototype here:   [UNcover Bounty Tasks](https://projects.invisionapp.com/share/7TUN9MQY259#/screens/391426174)

You will need to request access permission to access inspect mode on the InVision prototype. 

Screenshot of intended design also attached. 

*Please note there may be some inconsistencies with the InVision prototype and the live CSS. If this occurs, please use the styles defined by the exisiting CSS. 

# Get Up & Running

Please check the README and CONTRIBUTING files:

https://github.com/cennznet/explorer/blob/master/portal/README.md https://github.com/cennznet/explorer/blob/master/portal/CONTRIBUTING.md 

# Guidelines & Hints

The main reason why the information looks poor on mobile is because it was coded up using an HTML Table Tag. 

We do not want to change the HTML structure. 

The main thing to do here is to add a horizontal scrollbar and make the table data scrollable.

We also want to always show the scrollbar, so the user knows in advance that there is more data to be seen.

The assets page is located at: views/pages/token-list.njk.

The current scrollbar is implemented on views/pages/tx-list.njk (look for className scroll-bar-wrapper).

The code to handle the scrollbar is located at: src/assets/js/mobile/scroll-bar.js (you should not have to change this).

Add any other necessary mobile css to _mobile.scss to reflect the design (hint: see className mobile-wrapper-tx-table.

The transaction’s page (pages/tx-list.njk) filter controls have already been styled so please refer there for   guidance on how to style the assets page’s filter controls.

# Test & Verify Your Changes 

## Mobile Devices 

Ensure your changes work on:

- The latest Android and iPhones phones.

- Chrome, Safari and FireFox responsive views.

Ensure your changes work as expected and look exactly like the attached design.

Also ensure that the changes do not introduce bugs or break the desktop version.

# Expected Outcome 

Given user navigates to assets page using mobile device 

When user navigates to the assets section 

Then user should see symbol 

And user should see volume

And user should see decimal

And user should see type

Given  user navigates to assets page using mobile device 

When user navigates to the assets filter section 

And user clicks on Assets or Type filter 

Then user should see native scrollbar 

Given user navigates to assets page using mobile device 

When user navigates to the assets filter section 

And user selects filter 

Then user should see filtered results 

Given user navigates to assets page using desktop 

When user navigates to the assets section 

Then user should see standard full screen UI

## If you have any questions, please don’t hesitate to contact our team on [Slack](https://join.slack.com/t/centralitydev/shared_invite/enQtNjk5NjE1MDI5NjgzLWJlOTM2NjMxNTMyZDY0OGM0MjY2YjQ1YjJkNTk2ZTIzNTFmYmM3OTIwZWE3NzNhMzE4Mjg4MmY1YzBiZDk0ZDE).

Current screenshots
image

–––––––––––––––Designed Screen (how your design should look)
Assets

### PR DESCRIPTION
**Pull request by mul53**
Closes #17

### CODE REVIEW

**Comment by mul53** [comment]
I need more clarification on the filter assets and registered assets, you can review the work already done. So for filter assets i used the `filteredTotal` value and for registered assets i used the `total` value

File: portal/src/assets/style/_mobile.scss [hunk 1]
```diff
@@ -825,6 +832,62 @@ hr {
...
+
+    hr {
+      width: initial;
+      margin-left: 1rem;
+      margin-right: 1rem;
+      height: 1px;
+    }
+  }
+
+  .mobile-margin-x {
```
Comment (nickjohngray) [inline portal/src/assets/style/_mobile.scss]: Please give a better name , what is X?

File: portal/views/pages/token-list.njk [hunk 2]
```diff
@@ -5,23 +5,33 @@
...
+      <hr>
+      <div class="block-section__sub-header block-section__header--mobile-row">
+        <strong>{{ _('page.tokenList.registeredAssets') }}:</strong> ({{ total | numberFormat }} {{ _('helper.field.total') }})
+      </div>
+      <div class="scroll-bar-wrapper mobile-margin-x">
+        <div class="scroll-bar-div"></div>
+      </div>
       <div class="filter-result">
-        <div class="filter-result__result">
+        <div class="filter-result__result mobile-text-left mobile-margin-x mobile-margin-y-none">
```
Comment (nickjohngray) [inline portal/views/pages/token-list.njk]: Please give a better name for mobile-margin-y-none. Maybe top ?

File: portal/src/assets/style/_mobile.scss (hunk 1 above)
Comment (mul53) [reply portal/src/assets/style/_mobile.scss]: x-axis, from left to right. But i will give it a  better name

**Comment by mul53** [comment]
@nickjohngray i have made the classnames more readable

**Pull request merged**